### DISCUSSION TITLE
Idea for an option

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#6133] [Ideas] [answered]
As an owner of several Scratch accounts, I often have the need to check them and it's annoying having to click on my profile picture, press "Sign Out", then sign back in, and do whatever, so my idea is add some button to say, "Sign Into Different Account", which is a shorter way of doing the original process.

**Comment by Samq64** [comment, marked as answer]
See #1074 and #3327
